plugin-testing-framework/definitions/replace.d.ts

```diff
@@ -0,0 +1,13 @@
+interface ReplaceOptions {
+	regex: string;
+	replacement: string;
+	paths: Array<string>;
+	recursive: boolean;
+	silent: boolean;
```

Comment:
If its retrieved from DefinitelyTyped (I don't think it is), we should have included that file header that is on every d.ts file there

```
// Type definitions for xxx
// Project: xxx
// Definitions by: xxx
// Definitions: xxx
```